### ISSUE TITLE
metadata : scroll infini des histogrammes

### ISSUE THREAD

**Issue by paulgirard** [medialab/toflit18#101]

**Comment by gdaudin** [comment]
@paulgirard Est-ce que c’est d’actualité ?

**Comment by gdaudin** [comment]
Oui, ce serait cool... http://toflit18.medialab.sciences-po.fr/#/exploration/meta?model=product&dataType=product_orthographic

**Comment by gdaudin** [comment]
Ok, cela marche en effet. J’ai changé le texte du bandeau de la viz pour enlever l’allusion à la limite du nombre d’items dont on peut donner les chronologies.
